### ISSUE TITLE
Emu Plugin Not Importing Adversaries After Installation

### ISSUE THREAD

**Issue by mllamazares** [mitre/emu#43]
**Title:**  
Emu Plugin Not Importing Adversaries After Installation

**Description:**  
After following the [Emu installation instructions](https://github.com/mitre/emu?tab=readme-ov-file#installation), the plugin dashboard does not display any new adversaries, and the `plugins/emu/data` folder remains empty.

## Steps to Reproduce

1. Install the Linux dependency mentioned in [Emu Additional Setup](https://github.com/mitre/emu?tab=readme-ov-file#additional-setup): `apt-get install zlib1g`
2. Install Caldera [using Docker](https://caldera.readthedocs.io/en/latest/Installing-Caldera.html#docker-deployment):
    ```shell
    git clone https://github.com/mitre/caldera.git --recursive
    cd caldera
    docker build --build-arg WIN_BUILD=true . -t caldera:server
    ```
3. Enable the Emu plugin by adding `- emu` to the list of enabled plugins in `conf/default.yml` (for running Caldera in insecure mode).
4. Start the Caldera server: `docker run -p 7010:7010 -p 7011:7011 -p 7012:7012 -p 8888:8888 caldera:server -E default`
5. Stop the server.
6. Run the `download_payloads.sh` script.
7. According to the [Emu installation instructions](https://github.com/mitre/emu?tab=readme-ov-file#installation), the Adversary Emulation Library should be downloaded to the data folder at this point, but it remains empty.
8. Restart Caldera, open the `emu` tab, and confirm that no adversaries have been imported (as shown in the screenshot below).

![image](https://github.com/user-attachments/assets/0881d823-4da8-4c22-ad96-9c6a06faaa6d)

## Desktop Info
- OS: Linux Ubuntu
- Browser: Firefox
- Version: 131.0.3
- Caldera Version: 5.0.0

**Comment by mjmcphee** [comment]
same exact issue, with exception I have attempted to run in Ubuntu image and not Docker container over the top.

Attempted on versions 22.04, 24.04.

**Comment by marcosmcampos** [comment]
Same issue here. Can repro easily.

**Comment by marcosmcampos** [comment]
I tried removing emu and starting caldera again, even --build again to see if anything changes.

I fixed my issue by running doing the following:

1. Stop caldera (if running)
2. `rm -Rf adversary-emulation-plans ` (located inside /caldera/plugins/emu/data
3. Running caldera

**Comment by handler0ne** [comment]
@mrfieldsoncyber your UPDATE did it for me. Had the same issue, but could resolve it thanks to your fix!

**Comment by mllamazares** [comment]
@mrfieldsoncyber thanks for that. Did you manage to replicate that fix with the Docker installation?

Here's the explanation of the fix: https://github.com/mitre/caldera/issues/3035#issuecomment-2318271919

#### TL;DR  
The hook that is triggered when the plugin starts checks if the folder exists to download the adversaries. However, that folder is created by the `download_payloads.sh` executed in the Dockerfile; thus, the adversaries are never downloaded.
